os/board/artik05x/src/artik05x_tash.c

```diff
@@ -265,6 +265,15 @@ int board_app_initialize(void)
 		}
 	}
 #endif /* CONFIG_RAMMTD */
+#ifdef CONFIG_LIBC_ZONEINFO_ROMFS
```

Comment:
you don't have to mount additional romfs, just put contents inside of tools\fs folder.